.github/workflows/python-package.yml

```diff
@@ -48,6 +47,7 @@ jobs:
     # development-specific dependencies by differencing the env and dev-env yml files
     - name: Check normal environment import
       run: |
+        unset PROJ_LIB
```

Comment:
maybe add a comment here to state why this is needed?

Reply:
Done